sql/database.sql

```diff
@@ -2901,6 +2904,11 @@ INSERT INTO `layout_options` (`form_id`,`field_id`,`group_id`,`title`,`seq`,`dat
 INSERT INTO `layout_options` (`form_id`,`field_id`,`group_id`,`title`,`seq`,`data_type`,`uor`,`fld_length`,`max_length`,`list_id`,`titlecols`,`datacols`,`default_value`,`edit_options`,`description`,`fld_rows`) VALUES ('DEM', 'guardiancountry'  , '8', 'Country'  ,80, 26, 1,0,0, 'country', 1, 1, '', '', 'Country', 0);
 INSERT INTO `layout_options` (`form_id`,`field_id`,`group_id`,`title`,`seq`,`data_type`,`uor`,`fld_length`,`max_length`,`list_id`,`titlecols`,`datacols`,`default_value`,`edit_options`,`description`,`fld_rows`) VALUES ('DEM', 'guardianphone'  , '8', 'Phone'  ,90, 2, 1,20,63, '', 1, 1, '', '', 'Phone', 0);
 INSERT INTO `layout_options` (`form_id`,`field_id`,`group_id`,`title`,`seq`,`data_type`,`uor`,`fld_length`,`max_length`,`list_id`,`titlecols`,`datacols`,`default_value`,`edit_options`,`description`,`fld_rows`) VALUES ('DEM', 'guardianworkphone'  , '8', 'Work Phone'  ,100, 2, 1,20,63, '', 1, 1, '', '', 'Work Phone', 0);
+
+--Inserting data for sex,sexual orientation, gender identity categories, A5 items
+INSERT INTO `layout_options` (`form_id`,`field_id`,`group_id`,`title`,`seq`,`data_type`,`uor`,`fld_length`,`max_length`,`list_id`,`titlecols`,`datacols`,`default_value`,`edit_options`,`description`) VALUES ('DEM', 'gender_identity', '1', 'Gender Identity', 110,1,2,0,100,'gender_identity',1,1,'','N','Gender Identity');
+INSERT INTO `layout_options` (`form_id`,`field_id`,`group_id`,`title`,`seq`,`data_type`,`uor`,`fld_length`,`max_length`,`list_id`,`titlecols`,`datacols`,`default_value`,`edit_options`,`description`) VALUES ('DEM', 'sexual_orientation', '1', 'Sexual Orientation', 120,1,2,0,100,'sexual_orientation',1,1,'','N','Sexual Orientation');
+--End inserting data for sex,sexual orientation, gender identity categories, A5 items
```

Comment:
no need for the comment above

Reply:
And to make this more clear place it above where it shows in the layout (I would make these entries after the sex field), and change the seq in the items in that group to correspond with it showing after the sex field. And for now, I would not make these 2 items required (we can decide on that later).